### PR TITLE
Provide unique id

### PR DESCRIPTION
**Pull request by Santobert**

### CODE REVIEW

File: pybotvac/account.py [hunk 1]
```diff
@@ -24,6 +24,30 @@
...
+        "street_1": Any(str, None),
+        "street_2": Any(str, None),
+        "city": Any(str, None),
+        "post_code": Any(str, None),
+        "province": Any(str, None),
+        "state_region": Any(str, None),
+        "country_code": Any(str, None),
+        "source": Any(str, None),
+        "developer": Any(bool, None),
+        Required("email"): str,
```
Comment (chemelli74) [inline pybotvac/account.py:44]: Using my account, the returned json doesn't have an "email" key.

Simone

Comment (chemelli74) [inline pybotvac/account.py]: Using my account, the returned json doesn't have an "email" key.

Simone

```suggestion
        "email": Any(str, None),
```

**Review by chemelli74 — COMMENTED** [review]
Small change and it works fine.

Thx!

Simone

**Review by chemelli74 — APPROVED** [review]
LGTM 👍

**Comment by Santobert** [comment]
So we're ready for review then?

**Comment by Santobert** [comment]
@chemelli74 So we're ready for review then?

**Comment by chemelli74** [comment]
> @chemelli74 So we're ready for review then?

Yeah, works as expected. Already implemented in HoneAssistant ;-)

Simone

**Comment by Santobert** [comment]
@stianaske Could you please review this PR ?

**Comment by stianaske** [comment]
Looks good to me. Will merge and push to pypi.

**Pull request merged**